### PR TITLE
Initial implementation of cert resolver.

### PR DESCRIPTION
**Pull request by Blackbaud-BobbyEarl**
See https://github.com/blackbaud/skyux-sdk-cli/pull/32 for instructions on how to test.

### CODE REVIEW

File: cli/utils/cert-resolver.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+/*jslint node: true */
+'use strict';
+
+const fs = require('fs-extra');
+const os = require('os');
+const path = require('path');
+const logger = require('@blackbaud/skyux-logger');
+
+function getResolver() {
+  const certRootPath = path.resolve(`${os.homedir()}/.skyux/certs/`);
```
Comment (Blackbaud-SteveBrush) [inline cli/utils/cert-resolver.js]: This value is identical to the one in the CLI. Probably impossible, but do you think there's a way this value could live in one place? Maybe the path is passed into the `getResolver` function from the CLI?

Comment (Blackbaud-BobbyEarl) [reply cli/utils/cert-resolver.js]: Great suggestion!

**Comment by Blackbaud-BobbyEarl** [comment]
Re-running CI.

File: .eslintrc.json [hunk 2]
```diff
@@ -16,6 +16,7 @@
     "import/order": "off",
     "lines-around-directive": "off",
     "no-console": "off",
+    "no-else-return": "off",
```
Comment (Blackbaud-SteveBrush) [inline .eslintrc.json]: Ah man, I like this rule. 👎

Comment (Blackbaud-BobbyEarl) [reply .eslintrc.json]: Updated to not use this technique.

**Comment by Blackbaud-BobbyEarl** [comment]
Closing in favor of https://github.com/blackbaud/skyux-sdk-builder/pull/168

**Pull request closed without merging**